creator-node/src/routes/files.js

```diff
@@ -398,7 +407,7 @@ const getCID = async (req, res) => {
    */
   startMs = Date.now()
   try {
-    const queryResults = await models.File.findOne({
+    queryResults = await models.File.findOne({
```

Comment:
why make this change? it's never re-assigned

Reply:
needed this later to try the location on disk from db storagePath

Reply:
oh duh